{"uuid": "a6af9f90-17c6-4c7d-93f2-2554e2b76ed5", "children": ["bdf1d6b0-5dd0-4ad9-b0a0-2c70c5e35029"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 114\nCurrent browser version is 131.0.6778.109 with binary path C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#sessionnotcreatedexception\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x005DA813+48355]\n\t(No symbol) [0x0056C4B1]\n\t(No symbol) [0x00475358]\n\t(No symbol) [0x004961AC]\n\t(No symbol) [0x00491EF3]\n\t(No symbol) [0x00490579]\n\t(No symbol) [0x004C0C55]\n\t(No symbol) [0x004C093C]\n\t(No symbol) [0x004BA536]\n\t(No symbol) [0x004982DC]\n\t(No symbol) [0x004993DD]\n\tGetHandleVerifier [0x0083AABD+2539405]\n\tGetHandleVerifier [0x0087A78F+2800735]\n\tGetHandleVerifier [0x0087456C+2775612]\n\tGetHandleVerifier [0x006651E0+616112]\n\t(No symbol) [0x00575F8C]\n\t(No symbol) [0x00572328]\n\t(No symbol) [0x0057240B]\n\t(No symbol) [0x00564FF7]\n\tBaseThreadInitThunk [0x75D2FCC9+25]\n\tRtlGetAppContainerNamedObjectPath [0x77A682AE+286]\n\tRtlGetAppContainerNamedObjectPath [0x77A6827E+238]\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\.venv\\lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\.venv\\lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\.venv\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\.venv\\lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\.venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\.venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\tests\\conftest.py\", line 19, in driver\n    driver = webdriver.Chrome(service=service, options=options)\n  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\.venv\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 47, in __init__\n    super().__init__(\n  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\.venv\\lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 69, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\.venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 261, in __init__\n    self.start_session(capabilities)\n  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\.venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 362, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\.venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 454, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\SmiligenceHrAdmin\\.venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1753440889246, "stop": 1753440896158}], "start": 1753440889246, "stop": 1753440896443}